.github/workflows/lint.yml

```diff
@@ -3,12 +3,17 @@ name: Lint
 on:
   pull_request:
     paths:
-      - src/**
-      - .clang-format
+    - .clang-format
+    - .prettierrc.json
+    - src/**
```

Comment:
This should also include `types/**`, `npm/**`, and `samples/**` at a minimum

Reply:
I decided to just ask git to list all tracked files + all untracked files that are not git ignored. This list should stay up to date.

Reply:
Not sure what you mean, on my screen it's still not showing those directories

Reply:
I changed only format.py and forgot GH actions. Will fix.